setup.py

```diff
@@ -0,0 +1,19 @@
+import io
+from setuptools import setup
+
+
+with io.open('README.md', 'rt', encoding='utf8') as f:
+  readme = f.read()
+
+
+setup(
+  name='Latexify',
+  url='https://github.com/odashi/latexify_py',
+  py_modules=['latexify'],
+  license='Apache License 2.0',
+  author='Yusuke Oda',
+  description='Generates LaTeX math description from Python functions.',
+  long_description=readme,
+  python_requires='>=3.6',
```

Comment:
As `ast` library is frequently changing, we could not provide supports for far-newer versions. I think it is still compatible as of v3.8, `>=3.6, <=3.8` is good for now.

Reply:
Done.